src/cp.js

```diff
@@ -150,6 +150,14 @@ function cpdirSyncRecursive(sourceDir, destDir, currentDepth, opts) {
   } // for files
 } // cpdirSyncRecursive
 
+// Checks if cureent file was created recently
+function checkRecentCreated(sources, index) {
```

Comment:
It appears like relying on the parameter list is consistent with GNU `cp`, I must have been mistaken when I tried it